moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -375,6 +375,18 @@ bool ConfigurationFilesWidget::loadGenFiles()
   file.write_on_changes = 0;
   gen_files_.push_back(file);
 
+  // trajopt_planning_pipeline.launch
```

Comment:
Let's keep changes to the MSA out of this PR, please put them in a separate PR

Reply:
Yes they should be in another PR.